### ISSUE TITLE
Intermediate evaluation metrics

### ISSUE THREAD

**Issue by swarnim-j** [google-deepmind/tapnet#136]
I'm trying to run the TAPIR training from scratch with low GPU resources and am adhering to the training configurations specifications from the paper (with gradient accumulation as mentioned here: https://github.com/google-deepmind/tapnet/issues/132#issuecomment-2567534143), and would possibly not be able to complete the training run.

Would it be possible to get intermediate evaluation metrics on the TapVid Datasets from your run to compare and see how our run is progressing?

**Comment by yangyi02** [comment]
Hi @swarnim-j ,

Thank you for the patience. Here is the intermediate evaluation metrics on TAPVid-DAVIS strided evaluation for training a tapir model with 100K steps, hope it helps.

Average Jaccard:
<img width="1488" alt="Image" src="https://github.com/user-attachments/assets/1f9cd7f7-41b8-441b-9752-71c194f4e8d0" />

Position Accuracy: 
<img width="1483" alt="Image" src="https://github.com/user-attachments/assets/7c586690-66da-4d51-96e3-06bb935c6e8c" />

Occlusion Accuracy:
<img width="1489" alt="Image" src="https://github.com/user-attachments/assets/3b93e91c-a85e-4ebe-957b-17378bfeed40" />

**Comment by yangyi02** [comment]
Here are the intermediate metrics on TAPVid-DAVIS **query first** evaluation:

Average Jaccard:
![Image](https://github.com/user-attachments/assets/92cdd9b1-6cd5-4e92-a223-5868192c7147)

Position Accuracy:
![Image](https://github.com/user-attachments/assets/27a0c2cb-7ae5-4ce5-86e4-c194314b5277)

Occlusion Accuracy:
![Image](https://github.com/user-attachments/assets/0f9c72db-f2e2-45cf-a903-c3b1bac71230)
